lib/deref.js

```diff
@@ -295,6 +295,9 @@ module.exports = {
         resolvedSchema = concreteUtils.addOuterPropsToRefSchemaIfIsSupported(resolvedSchema, outerProperties);
       }
       if (resolvedSchema) {
+        if (schema.example) {
```

Comment:
@aman-v-singh OpenAPI specification also mentions `examples` at param level, we'll also need to handle them here.

https://spec.openapis.org/oas/v3.1.0#fixed-fields-9

Reply:
@aman-v-singh So is having examples already working? i.e. `test` here is value that's generated right?

Reply:
Yes, it is generating the `value` by taking the first value in `examples` as the `example:` parameter

Eg.: For this definition:- 
```
openapi: '3.0.0'
info:
  version: '1.0.0'
  title: 'Sample API'
  description: Buy or rent spacecrafts

paths:
  /spacecrafts:
    post:
      summary: Read a spacecraft
      parameters:
        - name: world
          in: query
          description: Direction to sort by 1
          schema:
            $ref: '#/components/schemas/SpacecraftId'
          examples:
            test:
              value: 'desc'
            test2:
              value: 'asc'
      responses:
        '200':
          description: The spacecraft corresponding to the provided `spacecraftId`
          content:
            application/x-www-form-urlencoded:
              schema:
                {}
            application/json:
              schema:
                {}
components:
  schemas:
    SpacecraftId:
      description: The unique identifier of a spacecraft
      enum:
        - 'asc'
        - 'desc'
      type: string
  securitySchemes:
    bearer:
      type: http
      scheme: bearer
security:
  - bearer: []
```
**This is generated**

```
{
    "item": [
        {
            "id": "9791043d-d0a6-4df7-a7db-273c3d8766c3",
            "name": "Read a spacecraft",
            "request": {
                "name": "Read a spacecraft",
                "description": {},
                "url": {
                    "path": [
                        "spacecrafts"
                    ],
                    "host": [
                        "{{baseUrl}}"
                    ],
                    "query": [
                        {
                            "disabled": false,
                            "key": "world",
                            "value": "desc",
                            "description": "Direction to sort by 1"
                        }
                    ],
                    "variable": []
                },
                "header": [
                    {
                        "key": "Accept",
                        "value": "application/x-www-form-urlencoded"
                    }
                ],
                "method": "POST",
                "auth": null
            },
            "response": [
                {
                    "id": "5bbbaa20-9a3b-44a9-b1ff-0e3fa303d437",
                    "name": "The spacecraft corresponding to the provided `spacecraftId`",
                    "originalRequest": {
                        "url": {
                            "path": [
                                "spacecrafts"
                            ],
                            "host": [
                                "{{baseUrl}}"
                            ],
                            "query": [
                                {
                                    "key": "world",
                                    "value": "desc"
                                }
                            ],
                            "variable": []
                        },
                        "header": [
                            {
                                "description": {
                                    "content": "Added as a part of security scheme: bearer",
                                    "type": "text/plain"
                                },
                                "key": "Authorization",
                                "value": "Bearer <token>"
                            }
                        ],
                        "method": "POST",
                        "body": {}
                    },
                    "status": "OK",
                    "code": 200,
                    "header": [
                        {
                            "key": "Content-Type",
                            "value": "application/json"
                        }
                    ],
                    "body": "{}",
                    "cookie": [],
                    "_postman_previewlanguage": "json"
                }
            ],
            "event": [],
            "protocolProfileBehavior": {
                "disableBodyPruning": true
            }
        }
    ],
    "event": [],
    "variable": [
        {
            "type": "string",
            "value": "/",
            "key": "baseUrl"
        }
    ],
    "auth": {
        "type": "bearer",
        "bearer": [
            {
                "key": "token",
                "value": "{{bearerToken}}"
            }
        ]
    },
    "info": {
        "_postman_id": "8945ff36-6503-4c8b-aa4c-639a4f139450",
        "name": "Sample API",
        "schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
        "description": {
            "content": "Buy or rent spacecrafts",
            "type": "text/plain"
        }
    }
}
```